files/en-us/web/javascript/reference/global_objects/set/index.html

```diff
@@ -137,7 +137,7 @@ <h3 id="Iterating_Sets">Iterating Sets</h3>
 // converting between Set and Array
 const mySet2 = new Set([1, 2, 3, 4])
 mySet2.size                    // 4
-[...mySet2]                    // [1, 2, 3, 4]
+mySet2                         // {1, 2, 3, 4}
```

Comment:
The original line is correct here as it's destructuring the set into an array:

```suggestion
[...mySet2]                    // [1, 2, 3, 4]
```